rapid7_insightvm_cloud/bin/icon_rapid7_insightvm_cloud

```diff
@@ -6,7 +6,7 @@ from sys import argv
 
 Name = "Rapid7 InsightVM Cloud"
 Vendor = "rapid7"
-Version = "1.0.0"
+Version = "2.0.0"
```

Comment:
@mrinehart-r7 If Aine is changing from string to date (seems like the underlying type is still the same) could this just be a patch version update?

Reply:
That's a good question, we've never actually ran into a scenario like that before. It's worth trying out on a sample plugin and seeing what happens.

Reply:
Thinking more about this... the string -> date could be backwards incompatible since different date formats could have been provided within the string input, and the date input only accepts RFC 8601 IIRC